### DISCUSSION TITLE
Convert mouse sensitivity from resolution to resolution

### DISCUSSION THREAD

**Discussion by imNaphy** [ppy/osu#29238] [Ideas]
Recently, I upgraded my laptop and went from 1920x1080 to 2560x1600, and I could notice right away that my sensitivity was way too small compared to my old rig. Can you make the mouse sensitivity work the same on all resolutions? (Maybe with a toggle in case people were accustomed already to a certain resolution)

**Comment by peppy** [comment]
try multiplying it by 1.48.

**Reply by imNaphy** [reply]
Why 1.48x specifically?

**Reply by peppy** [reply]
1600 / 1080, basic math
